### PR TITLE
fix($time): deprecate the property

### PR DESCRIPTION
**Pull request by mariusandra**
## Problem

People keep using this old `$time` property that was only set on a few events. That's because it looks legit:

image

## Changes

Makes it clear this is not a good idea.

image

The alternative isn't _that_ great or intuitive right now, but it's at least _correct_.

## How did you test this code?

Looked at the popup in the taxonomic filter. Had to fix an issue with descriptions not showing.

### CODE REVIEW

**Comment by benjackwhite** [comment]
Can we not make the proper timestamp value selectable instead? 

I think people are going to be really confused by this and saying "use hogql" feels like a cop out.

**Comment by mariusandra** [comment]
Oh absolutely! 💯

This is a cop-out, but I thought if something's bleeding, it's better to already apply a bandage, instead of blatantly waiting for a doctor to show up :).

Fixing this properly requires:
- Creating a new taxonomic pill called "Event metadata"
- Adding into it the options `uuid`, `distinct_id`, `timestamp`, `event` and `person_id`
- Fixing all the places in the frontend that use filters, including datatypes, value autocomplete and date comparisons
- Fixing all the places in the backend that use filters
- Fixing all the places in the plugin server that use filters (action matching)
- Fixing all the feature flag libraries that use filters in local evaluation (local evaluation in feature flags for distinct_id was asked recently)

Definitely doable and "easy", but not "quick" 😅

**Comment by pauldambra** [comment]
> Fixing this properly requires

There's a chance that if we materialise the timestamp column as a mat_timestamp column we could query it that way

Which would "simplify" the querying side of things but feels yucky 🤷

File: frontend/src/lib/components/DefinitionPopover/DefinitionPopoverContents.tsx [hunk 1]
```diff
@@ -132,18 +132,17 @@ function DefinitionView({ group }: { group: TaxonomicFilterGroup }): JSX.Element
...
+        (definition && 'description' in definition && definition?.description) ||
+        (definition?.name &&
+            (keyMapping.element[definition.name]?.description || keyMapping.event[definition.name]?.description))
+
     const sharedComponents = (
         <>
-            {hasTaxonomyFeatures &&
-                definition &&
-                'description' in definition &&
-                (hasTaxonomyFeatures && definition.description ? (
```
Comment (thmsobrmlr) [inline frontend/src/lib/components/DefinitionPopover/DefinitionPopoverContents.tsx:140]: redundant `hasTaxonomyFeatures `

**Review by thmsobrmlr — APPROVED** [review]

File: frontend/src/lib/components/DefinitionPopover/DefinitionPopoverContents.tsx (hunk 1 above)
Comment (mariusandra) [reply frontend/src/lib/components/DefinitionPopover/DefinitionPopoverContents.tsx:140]: yeah.. that's one of the reasons why I deleted these lines :D

Comment (thmsobrmlr) [reply frontend/src/lib/components/DefinitionPopover/DefinitionPopoverContents.tsx:140]: Oh man, I must have looked at the wrong section :D

**Pull request merged**